From 907a10c58ec52cf0292363f9c9e68db7a9c2115c Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 26 Jan 2021 11:34:17 +0200
Subject: [PATCH] [SW-0] ppboot: greco: align pci ids naming

Updated spec has changed PCI ID names from goya2 to greco. This patch
add change to align names in ppboot with updated spec.

Change-Id: I870d6ab3bb6cdb35cf3846602fcaf36ae633b88d
(cherry picked from commit 35ca1dc18b4cd1e4183f27af0951997a8d2ec426)
---

diff --git a/goya2_arc/common.c b/goya2_arc/common.c
index 782db03..86f860d 100644
--- a/goya2_arc/common.c
+++ b/goya2_arc/common.c
@@ -84,7 +84,7 @@
 	char product_str[PRODUCT_ID_STR_SIZE];
 
 	/* Default device id for addin card */
-	device_id = PCI_IDS_GOYA2_ADDIN;
+	device_id = PCI_IDS_GRECO_ADDIN;
 
 	/*
 	 * Efuse stores "product id + rev" as 7 chars of which
@@ -108,7 +108,7 @@
 	product_str[PRODUCT_ID_STR_SIZE - 2] = '\0';
 
 	if (strcmp(product_str, "HL1120") == 0)
-		device_id = PCI_IDS_GOYA2_M2;
+		device_id = PCI_IDS_GRECO_M2;
 
 	return device_id;
 }
